stdlib/2and3/hmac.pyi

```diff
@@ -0,0 +1,35 @@
+# Stubs for hmac
+
+from typing import Any, Callable, Optional, Union, overload
+from types import ModuleType
+import sys
+
+_B = Union[bytes, bytearray]
+
+# TODO more precise type for object of hashlib
+_Hash = Any
+
+if sys.version_info >= (3, 4):
```

Comment:
What's the point of the version check? Maybe I'm missing something, but I can't find any difference between the two alternatives below.